rstblog/builder.py

```diff
@@ -125,12 +128,37 @@ def get_default_template_context(self):
         }
 
     def render_template(self, template_name, context=None):
+        """
+        Render a template in the current context.
+
+        Args:
+            template_name (str): the path to the template you want, relative to
+                the Jinja2 template loader root.
+
+        Keyword Args:
+            context (Context or None): Extra context to pass in to the
+                rendering.
```

Comment:
Is "keyword args" generally used to refer to named arguments? I think of them as specifically being the **kw catch-all for arguments that aren't explicitly named in the function definition.